src/main/java/de/qabel/desktop/remoteFS/RemoteFSController.java

```diff
@@ -0,0 +1,155 @@
+package de.qabel.desktop.remoteFS;
+
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import de.qabel.core.crypto.CryptoUtils;
+import de.qabel.core.crypto.QblECKeyPair;
+import de.qabel.desktop.CellValueFactory.BoxFileCellValueFactory;
+import de.qabel.desktop.CellValueFactory.BoxObjectCellValueFactory;
+import de.qabel.desktop.exceptions.QblStorageException;
+import de.qabel.desktop.storage.*;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.TreeItem;
+import javafx.scene.control.TreeTableColumn;
+import javafx.scene.control.TreeTableView;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.ResourceBundle;
+import java.util.UUID;
+
+
+public class RemoteFSController implements Initializable {
+
+    private ImageView folderIcon = new ImageView(
+            new Image(getClass().getResourceAsStream("/folder.jpeg"))
+    );
+
+    private TreeItem<BoxObject> rootNode;
+    private List<BoxFolder> folders;
+
+    @FXML
+    private TreeTableView treeTable;
+    @FXML
+    private TreeTableColumn<BoxObject, String> nameColumn;
+    @FXML
+    private TreeTableColumn<BoxObject, String> sizeColumn;
+    @FXML
+    private TreeTableColumn<BoxObject, String> dateColumn;
+
+    @FXML
+    public void initialize(URL location, ResourceBundle resources) {
+
+
+        try {
+            BoxNavigation nav = createSetup();
+            uploadFilesAndFolders(nav);
```

Comment:
do we have to create fixture data over and over again in a persitent storage? (s3)